scripts/buildAlpaquita2.fish

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env fish
+# This is for clang16.0.6 builds on Alpaquita linux
+
+source ./scripts/lib/build.fish
+
+if test "$PARALLELISM" = ""
+  set -xg PARALLELISM 64
+end
+echo "Using parallelism $PARALLELISM"
+
+if test "$COMPILER_VERSION" = ""
+  set -xg COMPILER_VERSION clang16.0.6
+end
+echo "Using compiler version $COMPILER_VERSION"
+
+if test "$COMPILER_VERSION" = "clang16.0.6"
+  set -xg CC_NAME clang
+  set -xg CXX_NAME clang++
+else
+  set -xg CC_NAME gcc-$COMPILER_VERSION
+  set -xg CXX_NAME g++-$COMPILER_VERSION
+end
+
+if test "$OPENSSL_VERSION" = ""
+  set -xg OPENSSL_VERSION 3.1
+end
+echo "Using openssl version $OPENSSL_VERSION"
+
+set -l pie ""
+
+set -g FULLARGS $argv \
+ -DCMAKE_BUILD_TYPE=$BUILDMODE \
+ -DCMAKE_INSTALL_PREFIX=/ \
+ -DSTATIC_EXECUTABLES=On \
+ -DUSE_ENTERPRISE=$ENTERPRISEEDITION \
+ -DUSE_MAINTAINER_MODE=$MAINTAINER \
+ -DCMAKE_LIBRARY_PATH=/opt/openssl-$OPENSSL_VERSION/lib \
+ -DOPENSSL_ROOT_DIR=/opt/openssl-$OPENSSL_VERSION \
+ -DUSE_STRICT_OPENSSL_VERSION=$USE_STRICT_OPENSSL \
+ -DBUILD_REPO_INFO=$BUILD_REPO_INFO \
+ -DCMAKE_ASM_COMPILER_AR=/usr/bin/ar \
+ -DCMAKE_ASM_COMPILER_RANLIB=/usr/bin/ranlib \
+ -DCMAKE_C_COMPILER_AR=/usr/bin/ar \
+ -DCMAKE_C_COMPILER_RANLIB=/usr/bin/ranlib \
+ -DCMAKE_CXX_COMPILER_AR=/usr/bin/ar \
+ -DCMAKE_CXX_COMPILER_RANLIB=/usr/bin/ranlib
```

Comment:
Does this mean we use the GNU tools here instead of the tools shipped with clang/llvm?
Wondering if this should be changed to:
```
-DCMAKE_ASM_COMPILER_AR="/usr/bin/llvm-ar-16"
-DCMAKE_ASM_COMPILER_RANLIB="/usr/bin/llvm-ranlib-16" 
-DCMAKE_C_COMPILER_AR="/usr/bin/llvm-ar-16"
-DCMAKE_C_COMPILER_RANLIB="/usr/bin/llvm-ranlib-16" 
-DCMAKE_CXX_COMPILER_AR="/usr/bin/llvm-ar-16" 
-DCMAKE_CXX_COMPILER_RANLIB="/usr/bin/llvm-ranlib-16"
```
But I am not really an expert on this matter.

Reply:
Good point, will change and see if it works. Strangely enough, none of these are autodetected by cmake (as it should be), therefore I had to add something. But I would indeed prefer the llvm ones, if they are present in the image. Will check.

Reply:
None of these are present in the image. Maybe this is why they are not auto detected. Anyway, will leave it as it is for now.